src/site/_data/paths/identity.json

```diff
@@ -0,0 +1,53 @@
+{
+    "slug": "identity",
+    "cover": "/images/collections/identity.svg",
+    "title": "Identity",
+    "updated": "February 12, 2021",
+    "description": "Understand and build advanced authentication systems.",
+    "overview": "A curated collection of high-quality resources that will help you understand authentication basics and how to build your own authentication systems with modern APIs.",
+    "topics": [
+      {
+        "title": "Understand authentication basics",
+        "pathItems": [
+          "sign-in-form-best-practices",
+          "sign-up-form-best-practices"
+        ]
+      },
+      {
+        "title": "Build authentication systems with modern APIs",
+        "pathItems": [
+            "web-otp",
+            "change-password-url"
+        ]
+      },
+      {
+        "title": "Strenghten security with two-factor authentication and two-step verification",
+        "pathItems": [
+            "sms-otp-form"   
+        ]
+      },
+      {
+        "title": "Build advanced authentication systems",
+        "pathItems": [
+          {
+            "title": "Set up a biometric re-authentication functionality with WebAuthn",
+            "url": "https://developers.google.com/codelabs/webauthn-reauth/#0"
+          }
+        ]
+      },
+      {
+        "title": "Build authentication with identity federation",
+        "pathItems": [
+          {
+            "title": "Use OAuth 2.0 to access Google APIs",
```

Comment:
I requested this URL, but was wondering if it's okay to have "Google APIs" in the subject line. The content itself is a good overview of how OAuth 2.0 works in general, using Google (you can consider it as an example identity provider).  
Ideally we should have generic guidance on OAuth 2.0 and OpenID Connect without an affiliation to any particular services, but in reality there's dialects and helper libraries when adopting to identity providers. I think realistically this is an OK approach.